backstage/package.json

```diff
@@ -45,7 +45,8 @@
   },
   "resolutions": {
     "@types/react": "^17",
-    "@types/react-dom": "^17"
+    "@types/react-dom": "^17",
+    "swagger-ui-react": "5.10.5"
```

Comment:
Why do we need to specify the versions of these? Especially w.r.t `swagger-ui-react` which is pinned to an exact version